docs/_posts/2018-10-11-524-release.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: blogpost
+title:  "5.2.4 Release Announcement"
+section_id: blog
+date:   2018-10-11 12:00:00
+author: Open GEE Development Team
+---
+
+Hi Open GEE Community!
+ 
+We are happy to announce the official release of Open GEE 5.2.4!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+ 
+Enhancements include:
+
+**Progress indicators during gesystemmanager processing**. Added progress log messages to gesystemmanager while assets are being built.
+
+**New "Save and Build" file menu option**. Fusion UI now includes a new file menu option that allows users to save and build assets with one click, instead of clicking the save option and then the build option immediately after.
+
+**C++98 no longer supported**. Building open GEE using C++98 standard is deprecated and is no longer supported. C++11 syntax can be used.
+
+**New build parameter to redirect the build output**. Build output can now be optionally directed to a folder of your choosing instead of the current hard-coded defaults. See earth_enterprise/BUILD.md in source code for more details.
+
+**New experimental build cache folder parameter to the build**. This is an experimental option to potentially speed up builds. More work remains to be done before using this option in production builds. Use of this option is not recommended except for giving feedback and/or helping to address issues with Open GEE's build process using scons cache. See earth_enterprise/BUILD.md in source code for more details.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.4-2.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160004.html).
```

Comment:
I think the release page URL should end with 5.2.4-4.final instead 5.2.4-2.final.